stable/mariadb/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mariadb
-version: 7.0.1
+version: 7.0.2
```

```suggestion
version: 7.1.0
```
As it is a new feature and according to [semver](http://semver.org/):

> Given a version number `MAJOR.MINOR.PATCH`, increment the:
> - MAJOR version when you make incompatible API changes,
> - MINOR version when you add functionality in a backwards-compatible manner, and
> - PATCH version when you make backwards-compatible bug fixes.

Can you bump the minor version instead of the patch one?

---

Thx for this hint. Of course will I bump the minor if I also add the ability to set resource requests/limits for this test-runner.